src/linuxX64Main/kotlin/mu/ConsoleOutputAppender.kt

```diff
@@ -0,0 +1,15 @@
+package mu
+
+import platform.posix.fprintf
+import platform.posix.stderr
+
+object ConsoleOutputAppender : Appender {
+    override fun trace(message: Any?) = println(message)
+    override fun debug(message: Any?) = println(message)
+    override fun info(message: Any?) = println(message)
+    override fun warn(message: Any?) = println(message)
```

Comment:
isn't warn should also be printed to stderr?

Reply:
The stderr text stream is used for error messages only, whereas the stdout text stream is used for ordinary text messages (including warnings).

Reply:
I am ok with your decision, but usually warn are not regular messages: https://stackoverflow.com/a/4919110/411965